implementations/rust/ockam/ockam/src/forwarder.rs

```diff
@@ -0,0 +1,96 @@
+use crate::Context;
+use ockam_core::{Address, Any, LocalMessage, Result, Route, Routed, TransportMessage, Worker};
+use tracing::info;
+
+/// Alias worker to register remote workers under local names.
+///
+/// To talk with this worker, you can use the [`RemoteForwarder`](crate::RemoteForwarder)
+/// which is a compatible client for this server.
+#[non_exhaustive]
+pub struct ForwardingService {}
+
+impl ForwardingService {
+    /// Start a forwarding service. The address of the forwarding service will be
+    /// `forwarding_service`.
+    pub async fn create(ctx: &Context) -> Result<()> {
+        ctx.start_worker("forwarding_service", Self {}).await?;
+        Ok(())
+    }
+}
+
+#[crate::worker]
+impl Worker for ForwardingService {
+    type Context = Context;
+    type Message = Any;
+
+    async fn handle_message(
+        &mut self,
+        ctx: &mut Self::Context,
+        msg: Routed<Self::Message>,
+    ) -> Result<()> {
+        let forward_route = msg.return_route();
+        let payload = msg.into_transport_message().payload;
```

Comment:
What is expected to be in payload here?

Reply:
The client expects to receive the exact same payload it sends.

https://github.com/ockam-network/ockam/blob/ffadf0633adcae4d0a061fa322fb17dd15afe6e6/implementations/rust/ockam/ockam/src/remote_forwarder.rs#L87-L92